polkadot/runtime/westend/src/lib.rs

```diff
@@ -771,7 +771,11 @@ impl pallet_staking::Config for Runtime {
 }
 
 impl pallet_staking_ah_client::Config for Runtime {
+	type RuntimeOrigin = RuntimeOrigin;
 	type CurrencyBalance = Balance;
+	type AssetHubId = AssetHubId;
+	type SendXcm = crate::xcm_config::XcmRouter;
+	type MaxXcmTransactWeight = MaxXcmTransactWeight; // TODO: check if it's safe to reuse
```

Comment:
This is probably not what you want - this previously represented the max of (reserve|lease|notify_core_count).
With XCMv5 we don't need the backup weight at all. Westend system chains all already support XCMv5, so you can just use `None` for the backup weight.